doc/userguide/src/CreatingTestData/CreatingUserKeywords.rst

```diff
@@ -259,6 +259,20 @@ at the end the list variable gets all the leftover arguments that do not match
 other arguments. The list variable can thus have any number of items, even zero.
 
 .. sourcecode:: robotframework
+    *** Test Cases ***
+    Varargs with user keywords
+       @{argList}    Create List        argOne    argTwo    argThree   argFour    argTen
```

Comment:
We haven't used `camelCase` format with arguments or other code (except for Java examples) in the User Guide. In here `@{argList}` could be `@{arg list}` or preferably just `@{args}`. With the actual arguments, you could use `arg1` or `arg one`.

We've also used the optional `=` when assigning variables in the examples. In other words, this line should start with `@{args} =`.

Finally, examples should use four spaces as a separator consistently unless arguments are explicitly aligned.